tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -316,7 +310,7 @@ public function testGetContributionReturnFunctionality(): void {
     // update contribution with invoice number
     $params = array_merge($params, [
       'id' => $contributionID,
-      'invoice_number' => CRM_Utils_Array::value('invoice_prefix', Civi::settings()->get('contribution_invoice_settings')) . "" . $contributionID,
+      'invoice_number' => Civi::settings()->get('invoice_prefix') . $contributionID,
```

Comment:
I can see the original was copy/paste from elsewhere, hence the extra `. ""`. In other languages I've seen sometimes you need to do this type of thing to force string, since maybe a prefix like `0001` got output as just `1`, although that doesn't seem to be true in php. But maybe something along those lines, I just can't think what.

But specifically for this test, isn't the prefix always null and known to be such? Not a blocker just was probably also part of copy/paste in the original.

Reply:
@demeritcowboy yeah I guess if I'd thought it through further I would have removed it